### PR TITLE
VIS: let PeriodConverter handle datetime64 data

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
xref https://github.com/pandas-dev/pandas/issues/9053 (but doesn't solve it)

### CODE REVIEW

**Comment by jreback** [comment]
can you rebase / update

File: pandas/tests/plotting/test_datetimelike.py [hunk 1]
```diff
@@ -1449,6 +1449,17 @@ def test_overlapping_datetime(self):
         s2.plot(ax=ax)
         s1.plot(ax=ax)
 
+    @pytest.mark.xfail
```
Comment (jreback) [inline pandas/tests/plotting/test_datetimelike.py]: can you add a reason to the xfail (ref an issue)

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/tests/plotting/test_datetimelike.py (hunk 1 above)
Comment (jorisvandenbossche) [reply pandas/tests/plotting/test_datetimelike.py]: the issue is already linked, see comment on first line of test (or do you mean in the xfail message itself?)

Comment (jreback) [reply pandas/tests/plotting/test_datetimelike.py]: yes you normally provide a ``reason=`` in the xfail itself, which then prints out when the tests run

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
lgtm. pls add a whatsnew if you think appropriate (or not ok too).

**Pull request merged**